CIPs/CIP-59/tables/streamtypes.csv

```diff
@@ -1,3 +1,6 @@
-name,           code,           description,                                link
-Tile,           0,           A stream type representing a json document,    https://github.com/ceramicnetwork/CIP/issues/56
-CAIP-10 Link,   1,           Link blockchain accounts to DIDs,              https://github.com/ceramicnetwork/CIP/issues/15
+name,                     code,   description,                                        link
+Tile,                     0,      A stream type representing a json document,         https://github.com/ceramicnetwork/CIP/issues/56
+CAIP-10 Link,             1,      Link blockchain accounts to DIDs,                   https://github.com/ceramicnetwork/CIP/issues/15
+Model,                    2,      Describes a schema in ComposeDB, 
```

Comment:
I don't love reducing Models to just a schema, as two models could have identical schemas but still be semantically different.  How about "Describes a grouping of document in ComposeDB that share the same schema"?